apps/morph_check

```diff
@@ -193,7 +193,7 @@ if __name__ == '__main__':
     yaml_path = args.config
     SEPARATOR = '=' * 40
 
-    config = {'neuron_checks': [], 'options': []}
+    config = {'neuron_checks': [], 'options': {}}
```

Comment:
It should. `if-else` blocks don't have their own scope in python, so anything you declare in them exists from there on.